cas/cas.go

```diff
@@ -238,14 +245,48 @@ func (ds Store) WriteACI(r io.Reader) (string, error) {
 			BlobKey:    key,
 			AppName:    im.Name.String(),
 			ImportTime: time.Now(),
+			Latest:     latest,
 		}
 		return WriteACIInfo(tx, aciinfo)
 	}); err != nil {
 		return "", fmt.Errorf("error writing ACI Info: %v", err)
 	}
+
+	// The treestore uses acirenderer that needs cas.ResolveKey that will
+	// need the aciinfo already written in the db.
```

Comment:
I'd either remove it or at least put it after explaining you're not writing the treestore, something like "We still need to write aciinfo because..."